### PR TITLE
[WIP] Feat/address book

### PR DESCRIPTION
**Pull request by dbuarque**

### CODE REVIEW

File: src/components/AddressBook/index.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  _renderEmpty = () => {
+    const TemporaryEmptyComponent = styled.Text`
+      font-family: Rubik-Regular;
+      font-size: 16px;
+      padding: 40px;
+      color: white;
+    `
+    return (
+      <TemporaryEmptyComponent>Your Address Book list is empty. Add a new Contact or Account.</TemporaryEmptyComponent>
```
Comment (RenanCostaNascimento) [inline src/components/AddressBook/index.js]: Translations.

File: src/components/CancelButton/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import PropTypes from 'prop-types'
+
+import { CancelButton, CancelText } from './elements'
+
+const CancelButtonComponent = ({onCancel, navigation}) => {
+  const goBack = () => { navigation.goBack() }
+
+  return (
+    <CancelButton onPress={onCancel || goBack}>
+      <CancelText>CANCEL</CancelText>
```
Comment (RenanCostaNascimento) [inline src/components/CancelButton/index.js]: Translations.

File: src/components/CancelButton/index.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    </CancelButton>
+  )
+}
+
+CancelButtonComponent.propTypes = {
+  onCancel: PropTypes.func,
+  navigation: function (props, propName, componentName) {
+    if (!props['onCancel'] && !props['navigation']) {
+      return new Error(
+        'Please provide either an onCancel callback or the navigation object to CancelButton.'
```
Comment (RenanCostaNascimento) [inline src/components/CancelButton/index.js]: Translations.

File: src/scenes/Accounts/index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+
+import AddressBook from '../../components/AddressBook'
+
+// Mock state
+const accounts = [{
```
Comment (RenanCostaNascimento) [inline src/scenes/Accounts/index.js]: This was used during development only, right?

File: src/scenes/Contacts/Add/index.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+
+import NavigationHeader from '../../../components/Navigation/Header'
+import AddressForm from '../../../components/AddressBook/AddressForm'
+
+import { ADD } from '../../../utils/constants'
+
+export default class EditContact extends Component {
+  static navigationOptions = ({navigation}) => ({
+    header: <NavigationHeader title='ADD CONTACT' onBack={() => navigation.goBack()} />
```
Comment (RenanCostaNascimento) [inline src/scenes/Contacts/Add/index.js]: Translations.

File: src/scenes/Contacts/index.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  _loadContacts = async () => {
+    try {
+      const contacts = await getContactsFromStore()
+      this.setState({
+        contacts,
+        error: null
+      })
+    } catch (e) {
+      this.setState({
+        error: 'There was a problem loading the contacts. Please try again.'
```
Comment (RenanCostaNascimento) [inline src/scenes/Contacts/index.js]: Translations.

File: src/scenes/Contacts/index.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    const { contacts } = this.state
+    const { navigation } = this.props
+
+    return (
+      <AddressBook
+        items={contacts}
+        navigation={navigation}
+        reloadData={this._loadContacts}
+      >
+        <FloatingButton text='ADD CONTACT' onPress={this._onAddPress} />
```
Comment (RenanCostaNascimento) [inline src/scenes/Contacts/index.js]: Translations.

File: src/scenes/EditAddressBookItem/index.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+import ClearButton from '../../components/ClearButton'
+
+import getContactStore from '../../store/contacts'
+import { EDIT } from '../../utils/constants'
+
+export default class EditContact extends Component {
+  static navigationOptions = ({navigation}) => {
+    const { address } = navigation.getParam('item', {})
+    return ({
+      header: <NavigationHeader title='EDIT' onBack={() => navigation.goBack()} rightButton={(
```
Comment (RenanCostaNascimento) [inline src/scenes/EditAddressBookItem/index.js]: Translations.

**Comment by carlospnav** [comment]
This PR needs Multiple Accounts to be ready. It shouldn't go to production before that one.

**Pull request merged**